### ISSUE TITLE
Instance of 'query' has no 'outerjoin' member (no-member)

### ISSUE THREAD

**Issue by nioncode** [anybox/pylint_flask_sqlalchemy#2]
It would be great if the plugin would support the `outerjoin` member, currently pylint fails with:
> models.py:47:12: E1101: Instance of 'query' has no 'outerjoin' member (no-member)

Sample code:
```
db.session.query(Book, Data)
.outerjoin(
    Data,
    Data.book_id == Book.id),
)
.all()
```

**Comment by petrus-v** [comment]
Thanks to ask new feature, do you have any plan create a merge request ?

**Comment by nioncode** [comment]
No, sorry, I have no idea how this plugin works and don't have time at the moment to investigate this further.
Currently, I simply ignore the one line where I use outerjoin in pylint.
